### PR TITLE
Drop src/Test namespace

### PR DESCRIPTION
**Pull request by keradus**
this is one of longly-bothering me topic. we were pushing some test files to dist package :(

best to review commit by commit

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-/tests/AbstractDoctrineAnnotationFixerTestCase.php export-ignore
-/tests/Console/TestToolInfo.php export-ignore
-/tests/Differ/AbstractDifferTestCase.php export-ignore
-/tests/Differ/DummyTestSplFileInfo.php export-ignore
-/tests/Fixtures/ export-ignore
-/tests/Linter/AbstractLinterTestCase.php export-ignore
-/tests/Report/AbstractReporterTestCase.php export-ignore
-/tests/Test/AbstractFixerWithAliasedOptionsTestCase.php export-ignore
-/tests/Test/AbstractTransformerTestCase.php export-ignore
+/tests/ export-ignore
```
Comment (kubawerlos) [inline .gitattributes:1]: Let's put the line in the proper place within the list below.

Comment (keradus) [reply .gitattributes]: done

**Review by kubawerlos — APPROVED** [review]

**Comment by stof** [comment]
Weren't those meant to be exposed to allow third-party fixers to write their tests with the same testing infrastructure ?

**Comment by keradus** [comment]
wow, i found out this msg by accident - that's also why so slow. maybe best to report new issue rather than commenting on issue closed for few months, @stof ?

they were wrongly exposed as part of source, meaning all the testing things like PHPUnit started to became non-dev dependencies.

I'm open to discuss best way to expose best testing class for 3rd party fixers (maybe as separated view)

**Pull request merged**